src/components/AdminSettings.vue

```diff
@@ -71,11 +71,21 @@
 				<div>
 					<input id="demoserver" v-model="serverMode" type="radio"
 						name="serverMode" value="demo" class="radio"
-						:disabled="updating" @input="fetchDemoServers">
+						:disabled="updating || hostErrors.some(x => x)" @input="fetchDemoServers">
 					<label for="demoserver">{{ t('richdocuments', 'Use a demo server') }}</label><br>
 					<p class="option-inline">
 						<em>{{ t('richdocuments', 'You can use a demo server provided by Collabora and other service providers for giving Collabora Online a try.') }}</em>
 					</p>
+					<div v-if="hostErrors.some(x => x)" class="option-inline">
```

Comment:
It would probably be a bit cleaner to move `hostErrors.some(x => x)` to a dedicated data property like hasErrors to make the template a bit more readable

Reply:
In fact, that's what I wrote at first even the name was the same. :) But gave an error telling me that there was no var named hostErrors. (Maybe it wasn't generated yet? No idea.) So I went this way.